reference/managed-delivery/gitops.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: single
+title:  "Managed Delivery"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Spinnaker supports _GitOps_ workflows to create and update Managed Delivery configurations.
```

Comment:
Thoughts on switching to more simplistic language than `GitOps`, e.g. `Git integration` and `Git-based workflows`? GitOps can be a pretty loaded term depending on individual background and experience.

Reply:
That's an excellent question... I really don't know. On the one hand, I feel like, for as much as I agree with you on the term being overloaded, most people in the industry would have a general understanding of what "GitOps" means here and how it applies; on the other, I don't think we necessarily gain too much from using it other than buzzword compliance. Another alternative would be to make that link to one of the most common articles on the topic (like [this one](https://www.weave.works/blog/what-is-gitops-really) from WeaveWorks, who coined the term).

Reply:
I think seeing that WeaveWorks article further reinforces my gut feel that GitOps has too much conceptual baggage — anecdotally when I've used the term I've noticed folks assume a bunch of things (e.g. you're on K8s, or the definition is statically all there rather than having some dynamism, or you commit changes to spec files to kick off software deploys, etc etc). Generally I would say that Netflix and Managed Delivery don't really agree with the 'strict' definition of GitOps, as we would like software versions and other types of details to change out-of-band from version control.

Reply:
Fair points. I removed the references to GitOps.

Reply:
If we're dropping `GitOps` from the text, should the file get renamed too?

Reply:
Will do.